README.md

```diff
@@ -2,7 +2,7 @@
 
```

Comment:
What do you mean "old" way? That code was taken from a fresh (within the last 2 weeks) Expressive skeleton ...